### PR TITLE
Retire storybooks

### PR DESCRIPTION
**Pull request by eessex**
Follow up to https://github.com/artsy/force/pull/6474, deprecate storybooks.
Related to https://artsyproduct.atlassian.net/browse/PLATFORM-2880

### CODE REVIEW

File: src/DevTools/MockRelayRenderer.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ import {
   createMockNetworkLayer,
   createMockNetworkLayer2,
 } from "./createMockNetworkLayer"
+import { __webpack_require__ } from "webpack"
```
Comment (icirellik) [inline src/DevTools/MockRelayRenderer.tsx:21]: What does this do?

**Review by icirellik — APPROVED** [review]
💯 so much cleanup.

File: src/DevTools/MockRelayRenderer.tsx (hunk 1 above)
Comment (eessex) [reply src/DevTools/MockRelayRenderer.tsx:21]: Fix a type error, was undefinied... not sure why that was ok before, but this at least quiets down our linters.

**Comment by artsyit** [comment]
:rocket: PR was released in [`v28.10.0`](https://github.com/artsy/reaction/releases/tag/v28.10.0) :rocket:

**Pull request merged**